server/src/main/java/org/candlepin/model/Content.java

```diff
@@ -160,9 +149,30 @@ public Content() {
      *  The ID for this content
      */
     public Content(String id) {
+        this();
+
         this.setId(id);
     }
 
+    public Content(String id, String name, String type, String label, String vendor) {
+        this(id);
+
+        this.setName(name);
+        this.setType(type);
+        this.setLabel(label);
+        this.setVendor(vendor);
+    }
+
+    public Content(String name, String id, String label, String type, String vendor, String contentUrl,
```

Comment:
@Ceiu  we should be consistent with the order of the arguments. I like the above order, but as long as we are consistent, I would be fine.